devicetypes/smartthings/zll-rgbw-bulb.src/zll-rgbw-bulb.groovy

```diff
@@ -29,15 +29,30 @@ metadata {
 
         attribute "colorName", "string"
 
+        // Generic
```

Comment:
Are these all 4 space?

Reply:
They were. I pushed a commit that converts them to tabs